src/main/java/jenkins/branch/CustomOrganizationFolderDescriptor.java

```diff
@@ -1,221 +0,0 @@
-/*
- * The MIT License
- *
- * Copyright 2015 CloudBees, Inc.
- *
- * Permission is hereby granted, free of charge, to any person obtaining a copy
- * of this software and associated documentation files (the "Software"), to deal
- * in the Software without restriction, including without limitation the rights
- * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
- * copies of the Software, and to permit persons to whom the Software is
- * furnished to do so, subject to the following conditions:
- *
- * The above copyright notice and this permission notice shall be included in
- * all copies or substantial portions of the Software.
- *
- * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
- * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
- * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
- * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
- * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
- * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
- * THE SOFTWARE.
- */
-
-package jenkins.branch;
-
-import hudson.Extension;
-import hudson.ExtensionList;
-import hudson.ExtensionListListener;
-import hudson.init.InitMilestone;
-import hudson.init.Initializer;
-import hudson.model.Descriptor;
-import hudson.model.DescriptorVisibilityFilter;
-import hudson.model.ItemGroup;
-import hudson.model.TopLevelItem;
-import hudson.model.TopLevelItemDescriptor;
-import hudson.model.View;
-import hudson.model.ViewGroup;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.logging.Level;
-import java.util.logging.Logger;
-import jenkins.scm.api.SCMNavigatorDescriptor;
-import jenkins.scm.impl.SingleSCMNavigator;
-import org.jenkins.ui.icon.IconSpec;
-import org.kohsuke.accmod.Restricted;
-import org.kohsuke.accmod.restrictions.NoExternalUse;
-
-/**
- * Ensures that when we have a non-null {@link SCMNavigatorDescriptor#newInstance(String)}, a special <b>New Item</b> entry is displayed instead of the generic {@link OrganizationFolder.DescriptorImpl}.
- */
-@SuppressWarnings("rawtypes") // not our fault
-@Restricted(NoExternalUse.class)
-public class CustomOrganizationFolderDescriptor extends TopLevelItemDescriptor implements IconSpec {
```

Comment:
This is kind of used here: https://github.com/jenkinsci/scm-api-plugin/blob/d01f419bc9dcc408e834cbc333b285b1c8058e69/src/main/java/jenkins/scm/api/SCMNavigatorDescriptor.java#L177 Are we safe if we remove this?

Reply:
Just means that `newInstance(String)` could be deprecated as unused along with other methods mentioned in https://github.com/jenkinsci/branch-api-plugin/pull/271#issue-724943402.